### ISSUE TITLE
Data is encrypted during view?

### ISSUE THREAD

**Issue by Datawords-HK** [Edwin-Luijten/strapi-encryptable-field#11]
Hello there,

I am having the same case like the picture shown in README:
https://raw.githubusercontent.com/Edwin-Luijten/strapi-encryptable-field/main/assets/role_based_decryption_overview.png

The encrypted field is still revealed as encrypted. Is it normal?
Could I have it decrypted and human readable?

Thanks.

**Comment by kmertozen** [comment]
did you fix it?

**Comment by Datawords-HK** [comment]
> ###
> [Jul 25, 2023](#issuecomment-1649260895)

No. I do hope it will be fixed. Or I have been requested to change to Laravel.

**Comment by kmertozen** [comment]
this may work
https://github.com/Edwin-Luijten/strapi-encryptable-field/pull/14

**Comment by prestop-max** [comment]
Having the same issue here while working with Content-type builder for components types, general encryption/decryption functionality does work for simple Collectiong type. Possible relation between these issues?

**Comment by thematrixdev** [comment]
Just tried 1.1.4 on Strapi 4.12.6. It seems to be working.
